File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = j = 0
        n = len(nums)
        while j < n:
            t = nums[j]
            while j < n and nums[j] == t: j += 1
            nums[i] = t
            i += 1
        return i

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        let mut i : usize = 0;
        let mut j : usize = 0;
        let n : usize = nums.len();
        while j < n {
            let temp : i32 = nums[j];
            while j < n && nums[j] == temp { j += 1; }
            nums[i] = temp;
            i += 1;
        }
        return i as i32;
    }
}